packages/plan-form/src/lang.ts

```diff
@@ -169,3 +169,15 @@ export const PLAN_TITLE_PLACEHOLDER = i18n.t("Enter the Plan's name");
 export const DESCRIPTION_PLACEHOLDER = i18n.t('Enter the plans Description');
 export const DYNAMIC_VALUE_LEGEND_TITLE = i18n.t('Dynamic Value');
 export const PATH_LABEL = i18n.t('Path');
+export const YES = i18n.t('yes');
+export const NO = i18n.t('no');
+export const SETTING_STATUS_TO_DRAFT = i18n.t('Are you sure, status will be set to draft');
+export const SETTING_STATUS_TO_ACTIVE = i18n.t(
+  "Are you sure, you won't be able to change the status back to draft"
```

Comment:
This sentence seems weird kidogo.  Should we have a questions mark after the word "sure"?

Reply:
@moshthepitt doing this here: https://github.com/OpenSRP/web/pull/496